### ISSUE TITLE
Is the data adjusted for splits/ bonus / dividends?

### ISSUE THREAD

**Issue by vsdubey** [swapniljariwala/nsepy#140]
Hi
Is the historical data for equities that we fetch via here adjusted for splits/ bonus / dividends?
Regards

**Comment by arpitjain2602** [comment]
Hi! 
It is not adjusted for splits for sure.

**Comment by pradeep0037** [comment]
Yes, looks like it is not adjusted. For ex, calculating 200 SMA for AARTIIND is giving incorrect results. @swapniljariwala - can you please let us know if adding support for this is planned?
